scss/mixins/_transition.scss

```diff
@@ -1,13 +1,21 @@
 // stylelint-disable property-blacklist
 @mixin transition($transition...) {
-  @if $enable-transitions {
-    @if length($transition) == 0 {
-      transition: $transition-base;
-    } @else {
-      transition: $transition;
+  @if length($transition) == 0 {
+    $transition: $transition-base;
+  }
+
+  @if length($transition) > 1 {
```

Comment:
This could be a `@else if`?

Reply:
It is fine in most cases, but `@if` can catch cases where invalid values ​​like `$transition-base: null, all 1s`.